.github/workflows/bazel.yml

```diff
@@ -614,7 +617,10 @@ jobs:
         uses: jfrog/setup-jfrog-cli@d0a59b1cdaeeb16e65b5039fc92b8507337f1559 # pin@v3
 
       - name: Set dry run environment variable
-        if: ${{ github.event_name != 'push' || github.repository_owner != 'magma' || github.ref_name != 'master' }}
+        if: |
+          ${{ github.event_name != 'push' || github.repository_owner != 'magma'
+          || github.ref_name != 'master' ||
+          startsWith(github.ref_name, 'v1.' ) == 'false' }}
```

Comment:
I think the logic here is wrong. This condition should be true if the branch **is not** master **and not** `v1.x`. But in the current state it will basically always be true because if it runs on master it is not `v1.` and on a release branch it is not master.

Reply:
So more like a `... || (github.ref_name != 'master' && startsWith(github.ref_name, 'v1.' ) == 'false') }}`?

Reply:
The logic looks right - not sure about the return value of `startsWith`, but we will see in the test run.

Reply:
Changed. `startsWith` returns a boolean, see the [Docs](https://docs.github.com/en/actions/learn-github-actions/expressions#startswith).